roc/convert_test.go

```diff
@@ -0,0 +1,41 @@
+package roc
+
+import (
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_go2cBool(t *testing.T) {
+	tests := []struct {
+		arg    bool
+		result uint
+	}{
+		{true, 1},
+		{false, 0},
+	}
+	for _, tt := range tests {
+		t.Run(strconv.FormatBool(tt.arg), func(t *testing.T) {
+			assert.Equal(t, tt.result, uint(go2cBool(tt.arg)))
+		})
+	}
+}
+
+func Test_go2cStr_c2goStr(t *testing.T) {
```

Comment:
Could we also add tests for just c2goStr and just go2cStr?

The current tests don't actually check zero-byte handling, it would be nice to cover that part more precisely.

Reply:
added